src/store/index.ts

```diff
@@ -40,8 +40,8 @@ if (typeof(host) === 'undefined') {
 }
 
 const defaultConfig = {
-    rendezvousURL: process.env['STAGE_MAILBOX_URL'],
-    transitRelayURL: process.env['STAGE_RELAY_URL'],
+    rendezvousURL: process.env['STAGE_MAILBOX_URL'] || 'ws://localhost:4000/v1',
```

Comment:
I see this link is used twice, so i suggest moving it to constants file if exists.